apycula/gowin_pack.py

```diff
@@ -15,7 +15,7 @@ def get_bels(data):
     for cell in data['modules']['top']['cells'].values():
         bel = cell['attributes']['NEXTPNR_BEL']
         row, col, num = belre.match(bel).groups() 
-        yield (cell['type'], int(row), int(col), num, cell['parameters'])
+        yield (cell['type'], int(row), int(col), num, cell['parameters'], cell['attributes'])
```

Comment:
This is getting a bit out of hand. Maybe we should make a named tuple or dataclass? Named tuple is probably the least refactoring. https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
done